src/main/java/forceitembattle/util/ItemBuilder.java

```diff
@@ -16,6 +21,7 @@
 
 public class ItemBuilder {
 
+    @Getter
     private transient ItemStack itemStack;

```

Comment:
Do we serialize the ItemBuilder?

Reply:
No actually thats useless there, I removed the transient, also added final and moved the lombok (next commit)